changelog.txt

```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.65 - released xxxx 2021
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* CorsixTH is now available as a Flatpak! (thanks to @rmnvgr !)
+* On MacOS DMG files are now supported as disk images for Theme Hospital data
+* Hospital policies will now carry over to your next level in the campaign
+* You can now remove destroyed rooms for a fee (config option)
+* The information dialogs that appear when a room is first built can be
+  disabled in the configuration file
+* Fire extinguishers in rooms now have a chance to save a smoking machine
+* More adviser messages added
+* New trophy added
+* Windows and room size now impact staff happiness
+* Roujin's challenge cheat is back
+
+# Changes
+* Emergencies with unknown diseases will no longer occur
+* The VIP system has been overhauled to provide more balanced ratings
+* Overpriced/Underpriced treatments now scale with difficulty
+* Placing a receptionist on a desk will now call her to the desk
+* Boiler breakdowns now depend on handymen
+* Faxes can no longer be opened while paused (unless editing while paused is
+  enabled)
+* Improvements to receptionist announcement frequency and behaviour
+* The game will now check common locations for Theme Hospital on startup if it
+  has not been configured
+* Games compiled without movie/audio can no longer access these settings
+
+# Translations
+* Greek language added! (thanks to dimitrios-iliadis!)
+* Updates to Brazilian-Portuguese language
+* Updates to English language
+* Fixes to Chinese unicode encoding
+
+# Bug Fixes
+* Issues scheduling emergencies have been resolved
+* Games will no longer crash on load if the jukebox was open
+* Games will no longer crash because of an empty music folder
+* Fixed a chance of Doctors getting stuck in a queue after editing a room
```

Comment:
`Fixed a possibility of Doctors getting stuck in a queue after editing a room` ?